web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -951,6 +951,56 @@ span.upsell-price .tax {
   min-height: 0;
 }
 
+.front-page-luova-block {
+  background-image: url("../assets/LuovaBanner.png");
+  margin-top: -15px;
+  height: 350px;
+  border-bottom: 1px solid #FFFFFF;
+
+  h2 {
+    color: #FFFFFF;
+    font-weight: bolder;
+    line-height: 1.25em;
+    font-size: 3.5em;
+  }
+
+  p {
+    color: #FFFFFF;
+    font-size: 20px;
+    padding-right: 150px;
+  }
+
+  a:hover {
+    color: #49BCFB;
+  }
+
+  .btn {
+    margin-top: 15px;
+    margin-left: 0px;
+    background-color: #00B0FF;
+    color: #FFFFFF;
+    font-size: 1.2em;
+    padding: 4px 10px;  
+  }
+
+  @media (max-width: $xl) {
+    background-image: url("../assets/LuovaBanner2.png");
```

Comment:
Let's give these assets better names. You could do something like LuovaBanner1500x350.png - where the 1500x350 is the size in px of the file.